magefile.go

```diff
@@ -22,18 +22,26 @@ import (
 
 	// mage:import operator
 	"github.com/adobe/kratos/mage/operator"
+
+	// mage:import integrationTests
+	"github.com/adobe/kratos/mage/integrationTests"
 )
 
-// Cleans project
+// Clean Cleans project
 func Clean() {
 	//mg.Deps(operator.Clean)
 	mg.Deps(operator.Clean, helm.Clean)
 }
 
-// Build code, docker image and helm chart
+// BUild Builds code, docker image and helm chart
 func Build() {
 	mg.SerialDeps(operator.Build)
 	mg.SerialDeps(helm.PackageChart)
 	mg.SerialDeps(dockerutils.BuildImage)
 	mg.SerialDeps(dockerutils.PushImage)
 }
+
+// RunIntegrationTests Runs Integration Tests using KUTTL
+func RunIntegrationTests() {
```

Comment:
we already have `integrationtests:runIntegrationTests`, do we need this target as well?

Reply:
removed the namespace and target